.github/workflows/devcontainer.yaml

```
@@ -43,25 +46,17 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      - id: set-options
-        run: |
-          if [ "${{ github.ref }}" = "refs/heads/main" ]; then
-            echo "push=true" >>$GITHUB_OUTPUT
-          else
-            echo "push=false" >>$GITHUB_OUTPUT
-          fi
-
       - name: Build
         uses: docker/build-push-action@v4
         with:
           file: "./.devcontainer/base-image/Dockerfile"
           tags: ${{ steps.meta.outputs.tags }}
           # TODO: add linux/arm64
           platforms: linux/amd64
-          push: ${{ steps.set-options.outputs.push }}
+          push: ${{ github.ref_name == 'main' && 'true' || 'false' }}
```

I often see the following. How about that?

```suggestion
          push: ${{ github.ref == 'refs/heads/main'}}
```

---

We can try; I'm not sure whether it will interpret 0 and 1 correctly...